xsimdConfig.cmake.in

```diff
@@ -18,6 +18,9 @@
 @PACKAGE_INIT@
 
 if(NOT TARGET @PROJECT_NAME@)
-  include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
-  get_target_property(@PROJECT_NAME@_INCLUDE_DIRS xsimd INTERFACE_INCLUDE_DIRECTORIES)
+    include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
+    get_target_property(@PROJECT_NAME@_INCLUDE_DIRS @PROJECT_NAME@ INTERFACE_INCLUDE_DIRECTORIES)
+    if(${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.8)
+        target_compile_features(@PROJECT_NAME@ INTERFACE cxx_std_14)
+    endif()
```

Comment:
Why not getting the property from the target like we do for include dirs? I think of something like
```cpp
get_target_property(@PROJECT_NAME@_COMPILE_FEATURES @PROJECT_NAME@ COMPILE_FEATURES)
```
Also I wonder if it is necessary to check for the version of CMake. Maeby older versions will simply ignore properties they don't support?

Reply:
I added here to check on CMake version. If we don't check on CMake version it would indeed make more sense in the target itself. But then I don't see any reason why to extract a `xsimd_COMPILE_FEATURES`: the end user could do it itself if so desired.

Reply:
In fact, I would even argue that there is no need to extract `xsimd_INCLUDE_DIRS`.